### PR TITLE
use Map.get insteadof Array.indexOf because loop takes up too much time

### PR DESCRIPTION
**Pull request by PrUnSt**
Related issue: #XXXX

**Description**
use Map.get insteadof Array.indexOf because loop takes up too much time
performance

### CODE REVIEW

**Comment by PrUnSt** [comment]
I wonder why always use array instead of map in three.js

**Comment by Mugen87** [comment]
> I wonder why always use array instead of map in three.js

Because `Map` is not supported in older browsers. The files in `examples/js` still need to work in e.g. IE11. `Map` has only limited support in IE11.

File: examples/jsm/loaders/FBXLoader.js [hunk 1]
```diff
@@ -2886,11 +2886,21 @@ var FBXLoader = ( function () {
 			var yIndex = - 1;
 			var zIndex = - 1;
 
+			const indexMap = {}
```
Comment (Mugen87) [inline examples/jsm/loaders/FBXLoader.js:2889]: You have to change `FBXLoader` in `examples/js` instead and then generate the version in `examples/jsm` with a script. You execute it like so:
```
node utils/modularize.js
```
Besides, please use no ES6 features. So no `const` and no `for/of` loop.

**Comment by mrdoob** [comment]
Please use tabs.

**Pull request closed without merging**